omod/src/main/resources/config.xml

```diff
@@ -70,23 +70,39 @@
         <property>${project.parent.artifactId}.server.url</property>
         <defaultValue></defaultValue>
         <description>
-            The m2sys server url..
+            The m2sys cloud scanner url.
         </description>
     </globalProperty>
 
     <globalProperty>
         <property>${project.parent.artifactId}.server.user</property>
         <defaultValue></defaultValue>
         <description>
-            The m2sys server username.
+            The m2sys local (local to the clinic) server username.
+        </description>
+    </globalProperty>
+
+    <globalProperty>
+        <property>${project.parent.artifactId}.server.national-user</property>

```

Comment:
lets use a dot after national (national.user)

Reply:
There will be a difference with local-user then, but if it's not a problem I can change it.